model/database/common.yaml

```diff
@@ -59,6 +63,53 @@ groups:
       - ref: db.cosmosdb.consistency_level
         requirement_level:
           conditionally_required: If available.
+      - ref: error.type
+        examples: ["Microsoft.Azure.Cosmos.CosmosException", "Microsoft.Azure.Cosmos.CosmosOperationCanceledException"]
+
+  - id: attributes.db.cosmosdb.network.minimal
+    type: attribute_group
+    brief: 'Azure Cosmos DB Client network request level attributes'
+    stability: experimental
+    extends: attributes.db.cosmosdb.minimal
+    attributes:
+      - ref: network.protocol.name
+        stability: experimental
+        brief: Specifies the host protocol.
+        examples: ["https", "rntbd"]
```

Comment:
connection mode term would be confusing here because it is client side property and we make few `http` calls also in direct mode. IMO its ok, if customer doesn't undertand, what `rntbd` really is, attribute key clearly says that its a network protocol. OR we can use `tcp` term instead of `rntbd`, thats how we talk to the customers about `rntbd` calls.